File: JavaLibrary/src/main/java/com/flat/simplex/lexer/TokenGroup.java
package com.flat.simplex.lexer;

public class TokenGroup {

    private Token start;
    private Token end;

    public TokenGroup(Token start, Token end) {
        this.start = start;
        this.end = end;
    }

    public Token getStart() {
        return start;
    }

    public Token getEnd() {
        return end;
    }
}

File: SimplexApp/simplex/lang/lexer/TokenGroup.js
var Key;

class TokenGroup {
    constructor(start, end) {
        this.start = start;
        this.end = end;
    }

    getStart() {
        return this.start;
    }

    getEnd() {
        return this.end;
    }
}

module.exports = TokenGroup;
Key = require("simplex/lang/lexer/Token.js");